doc/doc-site/docs/off-chain/rust-sdk.md

```diff
@@ -1,3 +1,249 @@
 # Rust SDK
 
-More docs coming soon!
+The code for the Rust SDK lives in `/econia/src/rust/sdk`.
+This code provides programmatic access to Econia exchanges, in addition to offering an example to put it all together shown later in this document.
```

Comment:
How about?

"The SDK provides direct access to the Econia protocol, and comes with an example script described below."